specs/protocol/predeploys.md

```diff
@@ -347,6 +347,17 @@ Address: `0x4200000000000000000000000000000000000021`
 
 The `EAS` predeploy implements the `Ethereum Attestation Service` protocol.
 
+## RevenueSharer
+[Implementation](https://github.com/ethereum-optimism/optimism/blob/6e12d4b630cc1d3dbhttps://github.com/ethereum-optimism/optimism/blob/gk/revenue-sharing/packages/contracts-bedrock/src/L2/RevenueSharer.sol) 
```

Comment:
This hyperlink is broken

Reply:
[b2e130c](https://github.com/ethereum-optimism/specs/pull/115/commits/b2e130c7f8f244189edff8eda063b2a9ba8c8587)